ztag/schema.py

```diff
@@ -532,6 +532,31 @@ def ztag_zgrab2_transformed(service, results):
                              "the handshake.")
 }))
 
+ztag_mongodb = ztag_zgrab2_transformed(service="MongoDB", results=SubRecord({
+        "build_info": SubRecord({
+            "version": String(doc="Version of mongodb server"),
+            "git_version": String(doc="Git Version of mongodb server"),
+            "max_wire_version": Signed32BitInteger(),
+            "build_environment": SubRecord({
+                "dist_mod": String(),
+                "dist_arch": String(),
+                "cc": String(),
+                "cc_flags": String(),
+                "cxx": String(),
+                "cxx_flags": String(),
+                "link_flags": String(),
+                "target_arch": String(),
+                "target_os": String()})}),
+        "is_master": SubRecord({
+            "is_master": Boolean(),
+            "max_wire_version": Signed32BitInteger(),
+            "min_wire_version": Signed32BitInteger(),
+            "max_bson_object_size": Signed32BitInteger(),
+            "max_write_batch_size": Signed32BitInteger(),
+            "logical_session_timeout_minutes": Signed32BitInteger(),
+            "max_message_size_bytes": Signed32BitInteger(),
+            "read_only": Boolean()})}))
```

Comment:
**Nit**: I'd put each of the }) on their own line.

Reply:
I agree... thought I was following rest of file, but def not in code surrounding this block. Will change.